libs/javalib/api/src/mill/javalib/api/JvmWorkerUtil.scala

```diff
@@ -146,4 +150,15 @@ object JvmWorkerUtil {
       all.filter(v => v.nonEmpty && v >= versionParts.take(v.length)).map(_.mkString(".") + "-")
     (plus ++ minus).distinct.toSeq
   }
+
+  /**
+   * Checks whether the version of the scala-library should be `2.13.x` or match the given `scalaVersion`
+   * @param scalaVersion The Scala version
+   * @return `true` if the scala-library version should be a `2.13.`
+   */
+  def needsScala213Library(scalaVersion: String): Boolean = {
+    val sv = minorMajorVersion(scalaVersion)
+    // Some Dotty versions and all Scala 3 versions before 3.8
+    sv.major == 0 || (sv.major == 3 && sv.minor < 8)
+  }
```

Comment:
What about  2.12？seems 2.12 will now with a 2.13 std

Reply:
No, that's not what the code does. But maybe I can improve the method name? 

`enforceScala213Library` maybe?